<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="text_view">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="edit_sentence_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>171</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Выбрать предложение</string>
   </property>
  </widget>
  <widget class="QPushButton" name="open_file_btn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>190</y>
     <width>121</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Открыть файл</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
